internal/venice-common/src/main/java/com/linkedin/venice/common/VeniceSystemStoreUtils.java

```diff
@@ -52,4 +52,13 @@ public static boolean isUserSystemStore(String storeName) {
     VeniceSystemStoreType veniceSystemStoreType = VeniceSystemStoreType.getSystemStoreType(storeName);
     return veniceSystemStoreType != null && veniceSystemStoreType.isStoreZkShared();
   }
+
+  /**
+   * @return <code>true</code> is the input store is a meta system store;
+   *         <code>false</code> otherwise.
+   */
+  public static boolean isMetaStore(String storeName) {
```

Comment:
This method could be removed after making the fix work for both meta and davinci push status system store.

Reply:
As explained above, I will remove it when we are ready to expand the fix to push status store.